modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/wal/IgniteWalHistoryReservationsTest.java

```diff
@@ -403,28 +403,20 @@ public void testWalHistoryPartiallyRemoved() throws Exception {
 
         int entryCnt = 10_000;
 
-        IgniteEx ig0 = (IgniteEx)startGrids(2);
+        IgniteEx ig0 = startGrids(2);
 
-        ig0.cluster().active(true);
+        ig0.cluster().state(ClusterState.ACTIVE);
 
         IgniteCache<Integer, Integer> cache = ig0.cache("cache1");
 
         for (int k = 0; k < entryCnt; k++)
             cache.put(k, k);
 
-        GridTestUtils.runAsync(new Callable<Object>() {
```

Comment:
this test seems flaky at all and it already fixed in ggce and still erroneous in ai master.